_alerts/2017-07-12-disruptive-forks.md

```diff
@@ -0,0 +1,86 @@
+---
+## This file is licensed under the MIT License (MIT) available on
+## http://opensource.org/licenses/MIT.
+
+title: "Potential disruptive chain forks"
+shorturl: "disruptive-forks"
+active: true
+banner: "Warning: Bitcoin may be unsafe to use on August 1st (click here for details)"
+
+## 7/12 to 7/28: "warning" (orange); 7/29 until resolution: "alert" (red)
+bannerclass: "warning"
+---
+*Last updated: 2017-07-12 12:00 UTC.  This page will be updated when new
+information becomes available.  See the [list of updates][].*
+
+On 1 August 2017, Bitcoin confirmation scores may become unreliable for
+an unknown length of time.  This means that any bitcoins you receive
+after August 1st may disappear from your wallet at a later time or be a
+type of bitcoin that other people will not accept as payment.
+
+Once the situation is resolved, confirmation scores will either
+automatically return to their normal reliability or there will be two
+(or more) competing versions of Bitcoin.  In the former case, you may
+return to using Bitcoin normally; in the later case, you will need to
+take extra steps in order to begin safely receiving bitcoins again.
+
+This post currently describes what actions you can take to prepare for
+this situation.  Subsequent to August 1st, we will update this post as
+best we can with relevant information, but you are also advised to
+monitor other Bitcoin [news sites][] and [community resources][] for
+updates and to cross-check all information, as someone may attempt to
+spread false news in order to exploit the situation.
+
+Remember that you alone are responsible for the safety of your bitcoins,
+and that if you lose control of them for any reason, there is nothing
+the operators or contributors to this website can do to help you.
+
+## Before August 1st
+
+1. If you accept bitcoins as payments, we recommend that you stop
+accepting Bitcoin payments at least 12 hours before 00:00 UTC on August
```

Comment:
Good idea!  Actually, let me first see if I can find some freely-licensed javascript that will do the conversion to localtime in the browser.